spec/features/posts_spec.rb

```diff
@@ -16,4 +16,14 @@
     click_button "Create Post"
     expect(page).to have_content "can't be blank"
   end
-end
+
+  scenario "user can view all posts" do
+    post_one = FactoryBot.create(:post)
+    post_two = FactoryBot.create(:post, title: "foo", body: "bar" )
```

Comment:
If you add Faker having different data for each of your factories would be automatic. Check out the fake data section in your testing book